### PR TITLE
Add local and remote stream component

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/controls/Stream.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+export interface IStreamProps {
+    stream: MediaStream;
+    // TODO(yunsi): Need to figure out if we want
```
Comment (asinner) [inline src/components/controls/Stream.tsx]: add `className` prop

File: src/components/controls/RemoteStream.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import { Stream } from './Stream'
+
+export interface IRemoteStreamProps {
+    remoteStream: { [id: string]: MediaStream };
+}
+
+export interface IRemoteStreamState { }
+
+export class RemoteStream extends React.PureComponent<IRemoteStreamProps, IRemoteStreamState> {
```
Comment (asinner) [inline src/components/controls/RemoteStream.tsx]: Remove `RemoteStream` class, move this rendering logic to `Conference`

File: src/components/controls/LocalStream.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import { Stream } from './Stream'
+
+export interface ILocalStreamProps {
+    localStream: MediaStream;
+}
+
+export interface ILocalStreamState { }
+
+export class LocalStream extends React.PureComponent<ILocalStreamProps, ILocalStreamState> {
```
Comment (asinner) [inline src/components/controls/LocalStream.tsx]: Remove `LocalStream`, move rendering logic to `Conference`

File: src/components/controls/Stream.tsx (hunk 1 above)
Comment (niuyunsi) [reply src/components/controls/Stream.tsx]: Done

File: src/components/controls/RemoteStream.tsx (hunk 2 above)
Comment (niuyunsi) [reply src/components/controls/RemoteStream.tsx]: Done

File: src/components/controls/LocalStream.tsx (hunk 3 above)
Comment (niuyunsi) [reply src/components/controls/LocalStream.tsx]: Done

**Review by asinner — APPROVED** [review]

**Pull request merged**